framework/raven_qsub_command.sh

```diff
@@ -4,6 +4,16 @@ if test -n "$PBS_O_WORKDIR"; then
     cd $PBS_O_WORKDIR
 fi
 
+# conda definitions should be set up in raven/.ravenrc after installing with raven/scripts/establish_conda_env.sh
+module load raven-devel
+module load MVAPICH2/2.0.1-GCC-4.9.2
```

Comment:
gcc is not in raven-devel? Have we documented this in the manual for the users?

Reply:
This isn't adding GCC (GCC is already there); it's adding MPI, although you can't tell by the module name.

A good next action would be to add this library to the raven-devel module; however, it's only necessary to load on the distributed environment, and including it in the qsub_command files assures the user doesn't need to be aware of them.

Note especially this is very specific to Falcon, and does not translate to other computing clusters.